{"uid":"105eecbcc7ed1c47","name":"Bank account transfer for native user","fullName":"test.test_costumer_login.TestCostumerLogin#test_costumer_login_pege","historyId":"81073f16d66b974b0c2542bf9ff983f0","time":{"start":1692731310119,"stop":1692731315157,"duration":5038},"status":"broken","statusMessage":"selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\n  (Session info: headless chrome=115.0.5790.170); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x555a15a1c613 <unknown>\n#1 0x555a15748537 <unknown>\n#2 0x555a1574cade <unknown>\n#3 0x555a1574e340 <unknown>\n#4 0x555a1574e3fc <unknown>\n#5 0x555a15788bff <unknown>\n#6 0x555a15787d17 <unknown>\n#7 0x555a1577d1a9 <unknown>\n#8 0x555a157a7a32 <unknown>\n#9 0x555a1577cbc6 <unknown>\n#10 0x555a157a7bfe <unknown>\n#11 0x555a157bfd09 <unknown>\n#12 0x555a157a7803 <unknown>\n#13 0x555a1577b68b <unknown>\n#14 0x555a1577c42e <unknown>\n#15 0x555a159ddc28 <unknown>\n#16 0x555a159e1af7 <unknown>\n#17 0x555a159ec01c <unknown>\n#18 0x555a159e2723 <unknown>\n#19 0x555a159b1457 <unknown>\n#20 0x555a15a06628 <unknown>\n#21 0x555a15a067b7 <unknown>\n#22 0x555a15a15483 <unknown>\n#23 0x7fd6e6af6b43 <unknown>","statusTrace":"self = <test.test_costumer_login.TestCostumerLogin object at 0x7fd6b8b6fcd0>\n\n    @allure.title(\"Bank account transfer for native user\")\n    @allure.severity(\"NORMAL\")\n    @pytest.mark.smoke\n    def test_costumer_login_pege(self):\n        self.costumer_login_page.open()\n        self.costumer_login_page.is_open_browser()\n        self.costumer_login_page.navigate_to_costumer_login()\n        self.costumer_login_page.replenishment_deposit()\n>       self.costumer_login_page.withdrawn_sum()\n\ntest/test_costumer_login.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nfeature_pages/costumer_login_page.py:41: in withdrawn_sum\n    self.select_element_is_visibility(self.locator.AMOUNT_INPUT).click()\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click\n    self._execute(Command.CLICK_ELEMENT)\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute\n    return self._parent.execute(command, params)\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fd6b8ae7f40>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"stale element reference\",\"message\":\"stale element reference: stale elemen...\\\\n#21 0x555a15a067b7 \\\\u003Cunknown>\\\\n#22 0x555a15a15483 \\\\u003Cunknown>\\\\n#23 0x7fd6e6af6b43 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\nE         (Session info: headless chrome=115.0.5790.170); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nE       Stacktrace:\nE       #0 0x555a15a1c613 <unknown>\nE       #1 0x555a15748537 <unknown>\nE       #2 0x555a1574cade <unknown>\nE       #3 0x555a1574e340 <unknown>\nE       #4 0x555a1574e3fc <unknown>\nE       #5 0x555a15788bff <unknown>\nE       #6 0x555a15787d17 <unknown>\nE       #7 0x555a1577d1a9 <unknown>\nE       #8 0x555a157a7a32 <unknown>\nE       #9 0x555a1577cbc6 <unknown>\nE       #10 0x555a157a7bfe <unknown>\nE       #11 0x555a157bfd09 <unknown>\nE       #12 0x555a157a7803 <unknown>\nE       #13 0x555a1577b68b <unknown>\nE       #14 0x555a1577c42e <unknown>\nE       #15 0x555a159ddc28 <unknown>\nE       #16 0x555a159e1af7 <unknown>\nE       #17 0x555a159ec01c <unknown>\nE       #18 0x555a159e2723 <unknown>\nE       #19 0x555a159b1457 <unknown>\nE       #20 0x555a15a06628 <unknown>\nE       #21 0x555a15a067b7 <unknown>\nE       #22 0x555a15a15483 <unknown>\nE       #23 0x7fd6e6af6b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: StaleElementReferenceException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1692731308902,"stop":1692731308918,"duration":16},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"driver","time":{"start":1692731308918,"stop":1692731310118,"duration":1200},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"setup","time":{"start":1692731310118,"stop":1692731310119,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\n  (Session info: headless chrome=115.0.5790.170); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x555a15a1c613 <unknown>\n#1 0x555a15748537 <unknown>\n#2 0x555a1574cade <unknown>\n#3 0x555a1574e340 <unknown>\n#4 0x555a1574e3fc <unknown>\n#5 0x555a15788bff <unknown>\n#6 0x555a15787d17 <unknown>\n#7 0x555a1577d1a9 <unknown>\n#8 0x555a157a7a32 <unknown>\n#9 0x555a1577cbc6 <unknown>\n#10 0x555a157a7bfe <unknown>\n#11 0x555a157bfd09 <unknown>\n#12 0x555a157a7803 <unknown>\n#13 0x555a1577b68b <unknown>\n#14 0x555a1577c42e <unknown>\n#15 0x555a159ddc28 <unknown>\n#16 0x555a159e1af7 <unknown>\n#17 0x555a159ec01c <unknown>\n#18 0x555a159e2723 <unknown>\n#19 0x555a159b1457 <unknown>\n#20 0x555a15a06628 <unknown>\n#21 0x555a15a067b7 <unknown>\n#22 0x555a15a15483 <unknown>\n#23 0x7fd6e6af6b43 <unknown>","statusTrace":"self = <test.test_costumer_login.TestCostumerLogin object at 0x7fd6b8b6fcd0>\n\n    @allure.title(\"Bank account transfer for native user\")\n    @allure.severity(\"NORMAL\")\n    @pytest.mark.smoke\n    def test_costumer_login_pege(self):\n        self.costumer_login_page.open()\n        self.costumer_login_page.is_open_browser()\n        self.costumer_login_page.navigate_to_costumer_login()\n        self.costumer_login_page.replenishment_deposit()\n>       self.costumer_login_page.withdrawn_sum()\n\ntest/test_costumer_login.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nfeature_pages/costumer_login_page.py:41: in withdrawn_sum\n    self.select_element_is_visibility(self.locator.AMOUNT_INPUT).click()\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click\n    self._execute(Command.CLICK_ELEMENT)\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute\n    return self._parent.execute(command, params)\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fd6b8ae7f40>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"stale element reference\",\"message\":\"stale element reference: stale elemen...\\\\n#21 0x555a15a067b7 \\\\u003Cunknown>\\\\n#22 0x555a15a15483 \\\\u003Cunknown>\\\\n#23 0x7fd6e6af6b43 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\nE         (Session info: headless chrome=115.0.5790.170); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nE       Stacktrace:\nE       #0 0x555a15a1c613 <unknown>\nE       #1 0x555a15748537 <unknown>\nE       #2 0x555a1574cade <unknown>\nE       #3 0x555a1574e340 <unknown>\nE       #4 0x555a1574e3fc <unknown>\nE       #5 0x555a15788bff <unknown>\nE       #6 0x555a15787d17 <unknown>\nE       #7 0x555a1577d1a9 <unknown>\nE       #8 0x555a157a7a32 <unknown>\nE       #9 0x555a1577cbc6 <unknown>\nE       #10 0x555a157a7bfe <unknown>\nE       #11 0x555a157bfd09 <unknown>\nE       #12 0x555a157a7803 <unknown>\nE       #13 0x555a1577b68b <unknown>\nE       #14 0x555a1577c42e <unknown>\nE       #15 0x555a159ddc28 <unknown>\nE       #16 0x555a159e1af7 <unknown>\nE       #17 0x555a159ec01c <unknown>\nE       #18 0x555a159e2723 <unknown>\nE       #19 0x555a159b1457 <unknown>\nE       #20 0x555a15a06628 <unknown>\nE       #21 0x555a15a067b7 <unknown>\nE       #22 0x555a15a15483 <unknown>\nE       #23 0x7fd6e6af6b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: StaleElementReferenceException","steps":[{"name":"Open  browser and navigate to https://www.globalsqa.com/angularJs-protractor/BankingProject/#/login","time":{"start":1692731310119,"stop":1692731310551,"duration":432},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Login your account and checking welcome title","time":{"start":1692731310555,"stop":1692731313823,"duration":3268},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Replenishment deposit and checking balance","time":{"start":1692731313823,"stop":1692731315059,"duration":1236},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Transfer sum","time":{"start":1692731315059,"stop":1692731315157,"duration":98},"status":"broken","statusMessage":"selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\n  (Session info: headless chrome=115.0.5790.170); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x555a15a1c613 <unknown>\n#1 0x555a15748537 <unknown>\n#2 0x555a1574cade <unknown>\n#3 0x555a1574e340 <unknown>\n#4 0x555a1574e3fc <unknown>\n#5 0x555a15788bff <unknown>\n#6 0x555a15787d17 <unknown>\n#7 0x555a1577d1a9 <unknown>\n#8 0x555a157a7a32 <unknown>\n#9 0x555a1577cbc6 <unknown>\n#10 0x555a157a7bfe <unknown>\n#11 0x555a157bfd09 <unknown>\n#12 0x555a157a7803 <unknown>\n#13 0x555a1577b68b <unknown>\n#14 0x555a1577c42e <unknown>\n#15 0x555a159ddc28 <unknown>\n#16 0x555a159e1af7 <unknown>\n#17 0x555a159ec01c <unknown>\n#18 0x555a159e2723 <unknown>\n#19 0x555a159b1457 <unknown>\n#20 0x555a15a06628 <unknown>\n#21 0x555a15a067b7 <unknown>\n#22 0x555a15a15483 <unknown>\n#23 0x7fd6e6af6b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/example_test_e2e_bank_app/example_test_e2e_bank_app/feature_pages/costumer_login_page.py\", line 41, in withdrawn_sum\n    self.select_element_is_visibility(self.locator.AMOUNT_INPUT).click()\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":4,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"driver::0","time":{"start":1692731315609,"stop":1692731315664,"duration":55},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"severity","value":"NORMAL"},{"name":"feature","value":"Login page"},{"name":"tag","value":"smoke"},{"name":"parentSuite","value":"test"},{"name":"suite","value":"test_costumer_login"},{"name":"subSuite","value":"TestCostumerLogin"},{"name":"host","value":"fv-az986-197"},{"name":"thread","value":"2205-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.test_costumer_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["smoke"]},"source":"105eecbcc7ed1c47.json","parameterValues":[]}